### PR TITLE
Add external links

### PR DESCRIPTION
**Pull request by thewilkybarkid**

### CODE REVIEW

File: vendor-extra/JatsContentBundle/tests/EventListener/BuildView/ExtLinkListenerTest.php [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+        $this->assertSame($expected, $event->getTemplate());
+    }
+
+    public function templateChoiceProvider() : iterable
+    {
+        yield 'ext-link element' => [
+            <<<XML
+<ext-link xlink:href="bar" xmlns="http://jats.nlm.nih.gov" xmlns:xlink="http://www.w3.org/1999/xlink">
+    foo
```
Comment (giorgiosironi) [inline vendor-extra/JatsContentBundle/tests/EventListener/BuildView/ExtLinkListenerTest.php]: slightly confusing that the positive example is a relative link (`bar`) which will then be ignored  by https://github.com/libero/browser/pull/104/files#diff-663e4050bbaaab66f585fa7bf668af63R35

File: vendor-extra/JatsContentBundle/tests/EventListener/BuildView/ExtLinkListenerTest.php [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+        $this->assertEmpty($view->getArguments());
+        $this->assertEmpty($view->getContext());
+    }
+
+    public function nodeProvider() : iterable
+    {
+        yield 'no href' => ['<ext-link xmlns="http://jats.nlm.nih.gov">foo</ext-link>'];
+        yield 'relative link' => [
+            <<<XML
+<ext-link xlink:href="bar" xmlns="http://jats.nlm.nih.gov" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (giorgiosironi) [inline vendor-extra/JatsContentBundle/tests/EventListener/BuildView/ExtLinkListenerTest.php]: the fact that this example is on multiple lines adds one direction of variation, but the main point of this example is the `bar` value; make them all stand on multiple lines or on a single line, consistently?

File: vendor-extra/JatsContentBundle/tests/EventListener/BuildView/ExtLinkListenerTest.php [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+     */
+    public function it_sets_the_text_argument_and_href_attribute() : void
+    {
+        $listener = new ExtLinkListener($this->createDumpingConverter());
+
+        $element = $this->loadElement(
+            <<<XML
+<jats:ext-link xlink:href="http://example.com" xmlns:jats="http://jats.nlm.nih.gov"
+    xmlns:xlink="http://www.w3.org/1999/xlink">
+    foo <jats:italic>bar</jats:italic> baz
```
Comment (giorgiosironi) [inline vendor-extra/JatsContentBundle/tests/EventListener/BuildView/ExtLinkListenerTest.php]: may be a complex example, maybe split:
- a test for text argument and href attribute
- a test with inline elements inside the content like `<jats:italic>bar</jats:italic>`

File: vendor-extra/JatsContentBundle/tests/EventListener/BuildView/ExtLinkListenerTest.php (hunk 1 above)
Comment (thewilkybarkid) [reply vendor-extra/JatsContentBundle/tests/EventListener/BuildView/ExtLinkListenerTest.php]: Ah yeah, they should be absolute unless deliberately relative.

**Pull request merged**